codebase/superdarn/src.lib/tk/fitacf_v3.0/src/preprocessing.c

```diff
@@ -823,8 +826,19 @@ double ACF_cutoff_pwr(FITPRMS *fit_prms){
     ni = (ni > 0) ? ni :  1;
     cpc = cutoff_power_correction(fit_prms);
     min_pwr = min_pwr/ni * cpc;
-    if (min_pwr < 1.0) min_pwr = fit_prms->noise;
-
+    
+    
+    /*  Commenting the line below out as it causes problems with processing
+            BOREALIS data by setting Inf values to SNR as the search noise is not 
+            estimated. Amending it temporarily with the check
+            for "skynoise" data to be nonzero. This is a quick fix that allows 
+            BOREALIS data to be processed properly but also makes no impact 
+            on the older data. A more elaborate/universal solution to this issue 
+            should be developed for th enext release. */
+   
+    /*if (min_pwr < 1.0) min_pwr = fit_prms->noise;*/
+    if ((min_pwr < 1.0) && (fit_prms->noise != 0.0)) min_pwr = fit_prms->noise;
+    
```

Comment:
Just to avoid implying that this problem can only occur with the BOREALIS radars, would you mind if we rephrase this slightly?
Also, I think you meant to write "Amending it temporarily with the check for **search noise** data to be nonzero"

```suggestion
    /*  The commented line below causes Inf values of the SNR when the search noise 
        is zero. Therefore, we introduce an additional condition that the search noise 
        must be nonzero. This is a temporary fix that allows the affected data to be 
        processed properly while not affecting the processing of any other data. 
        A more universal solution to this issue should be developed for the 
        next release. */
   
    /*if (min_pwr < 1.0) min_pwr = fit_prms->noise;*/
    if ((min_pwr < 1.0) && (fit_prms->noise != 0.0)) min_pwr = fit_prms->noise;
    
```